modules/cnf-about-ptp-fast-event-notifications-framework.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-about-ptp-fast-event-notifications-framework_{context}"]
+= About the PTP fast event notifications framework
+
+DU applications can subscribe to PTP fast events notifications that are generated by {product-title} with the PTP Operator and `cloud-event-proxy` sidecar container deployed. An AMQP event bus is also required.
+
+The `cloud-event-proxy` sidecar container can access the same resources as the primary vRAN application without consuming any of the resources of the primary application. In addition, there is no significant latency when communicating between the primary and sidecar application.
+
+The fast events notifications framework uses a REST API for communication and is based on the O-RAN API specification. The framework consists of a publisher, subscriber, and an AMQ bus to handle communications between the publisher and subscriber applications. The `cloud-event-proxy` sidecar is a utility container that runs in a pod that is loosely coupled to the main application container on the DU node. It provides an event publishing framework that allows you to subscribe DU applications to published PTP events.
+
+A high level overview of the PTP fast events consumer workflow for DUs is as follows:
+
+DU application requests a subscription:: The DU sends an API request to the `cloud-event-proxy` sidecar to create a PTP events subscription. The `cloud-event-proxy` sidecar creates a new subscription resource.
+
+cloud-event-proxy sidecar creates the subscription:: The event resource is persisted by the `cloud-event-proxy` sidecar. The `cloud-event-proxy` sidecar container sends an acknowledgment with an id and URL location to access the stored subscription resource. The sidecar creates an AMQ messaging listener protocol for the resource specified in the subscription.
+
+DU receives the PTP events notifications:: The DU events consumer listens to the address specified in the resource qualifier. Messages are received and passed to the `cloud-event-proxy` sidecar events processor.
```

Comment:
Actually its the side car that listens to the events. 
The cloud-event-proxy at DU consumer side listens to the address specified.......
processed  and passed to the DU event consumer via rest api to the return url specified in  the subscription